src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -15,14 +15,14 @@
     <PackageReference Include="Umbraco.Code" Version="2.0.0" PrivateAssets="all" />
   </ItemGroup>
 
-  <!-- Force Windows to use ICU. Otherwise Windows 10 2019H1+ will do it, but older Windows 10 and most, if not all, Windows Server editions will run NLS -->
   <ItemGroup>
+    <!-- Force Windows to use ICU. Otherwise Windows 10 2019H1+ will do it, but older Windows 10 and most, if not all, Windows Server editions will run NLS -->
```

Comment:
Might as well update this comment, WS2022 always uses ICU on .NET 5.0 and later versions.

Reply:
You're totally right! .NET 5+ uses ICU by default, but that depends on whether the ICU native library can be loaded and that (by default) depends on whether the OS ships the library, as mentioned here: https://docs.microsoft.com/en-us/dotnet/core/compatibility/globalization/5.0/icu-globalization-api

> Windows 10 May 2019 Update and later versions ship with the ICU native library. If the .NET runtime can't load ICU, it uses NLS instead.

We can probably update this comment to be more generic, as we're actually opting-in to app-local ICU (except for MacOS) and bringing in the native libraries by referencing the `Microsoft.ICU.ICU4C.Runtime` package as documented here: https://docs.microsoft.com/en-us/dotnet/core/extensions/globalization-icu#app-local-icu

So that means both Windows and Linux will always use ICU and they are also pinned to a specific version, making culture specific things consistent on both. I'm not sure how MacOS deals with ICU and versioning though, but that would be good to investigate and document 😄

Reply:
I've updated the comment to be more generic about opting-in to app-local ICU 👍🏻 I'm curious why the condition for macOS is added though, would be great if someone with a Mac can test what happens if this condition is removed!